salsa_dancing_molecules/mean_square_displacement.py

```diff
@@ -1,29 +1,31 @@
 """Module for calculating the mean square displacement of a trajectory file."""
 
-from ase.io import read
+from ase.io.trajectory import Trajectory
 import numpy as np
+from .average import average
 
 
-def get_msd(traj_file, reference="final"):
+def get_msd(traj_file, t0, reference="final"):
     """
     Read trajectory file and call calculation function.
 
     arguments:
-        traj_file - trajectory file
-        reference - string or None
+        traj_file - str
```

Comment:
While you are here, it would be nice if you explain what the parameters mean, not only their type to improve the documentation.